docs/sphinx/user_guide/tutorial/matrix_transpose_local_array.rst

```diff
@@ -202,66 +202,90 @@ Here is the complete version including execution policy and kernel:
 in the lambda statement types. The ``RAJA::statement::Lambda`` type for each
-lambda shows the two ways to specify the local tile index arguments; we can 
-use an ``Offsets`` statement for each argument, or include multiple segment 
+lambda shows the two ways to specify the local tile index arguments; we can
+use an ``Offsets`` statement for each argument, or include multiple segment
 ids in one statement. Lastly, there is only one entry in the parameter
 tuple in this case, the local tile array. The placeholders in the
 previous example are not needed.
 
-.. note:: In this example, we need all five arguments in each lambda 
-          expression so the lambda expression argument lists are 
+.. note:: In this example, we need all five arguments in each lambda
+          expression so the lambda expression argument lists are
           the same. Another use case for the template parameter argument
           specification described here is to be able to pass only the
           arguments used in a lambda expression. In particular when we use
           multiple lambda expressions to represent a kernel, each lambda
           can have a different argument lists from the others.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-``RAJA::expt::launch`` Variants
+``RAJA::launch`` Variants
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``RAJA::expt::launch`` interface provides mechanisms to tile loops and use 
-*local arrays* in kernels to support algorithm patterns like the C-style kernel 
-above. When, using ``RAJA::expt::launch``, the ``RAJA_TEAM_SHARED`` macro is
-used to create a GPU shared memory array or a CPU stack memory array inside
-a kernel.
-
-``RAJA::expt::launch`` support methods for tiling over an iteration space
-using ``RAJA::expt::tile`` and ``RAJA::expt::loop_icount`` methods to tile
-loops and generate global iteration indices and local tile offsets.
-Moreover, lambda expressions for these methods will not be invoked for
-iterations outside the bounds of an iteration space when tile dimensions
-do not divide evenly the size of the iteration space; thus, no conditional
-checks on loop bounds are needed inside inner loops.
-
-A complete RAJA sequential CPU variant with kernel execution policy and 
+The ``RAJA::launch`` interface provides mechanisms to tile loops and use
+*local arrays* in kernels to support algorithm patterns like the C-style kernel
+above. When, using ``RAJA::launch``, the ``RAJA_TEAM_SHARED`` macro is
+used to create a GPU shared memory array when using the CUDA and HIP backends,
+static shared memory using SYCL is currently not supported. On the CPU allocating
+``RAJA_TEAM_SHARED`` corresponds to allocating memory on the stack.
+Alternatively, one can allocated dynamic shared memory by specifying the amount in
+the ``RAJA::LaunchParams`` struct. Dynamic shared memory is supported with all
+backends.
```

Comment:
It would be good to include what dynamic shared memory means on the CPU vs. GPU.